lms/views/decorators/h_api.py

```diff
@@ -5,6 +5,8 @@
 import functools
 import json
 import requests
+from requests import RequestException
+from requests import ReadTimeout
```

Comment:
Since `requests` in this module is now patched in the tests, these exception classes now need to be imported directly rather than accessed as `requests.*` otherwise they would be patched too (and patching exceptions does not work)